src/Theme/utils.ts

```diff
@@ -0,0 +1,55 @@
+import { DefaultTheme } from 'styled-components';
+
+/**
+ * Helper function that returns a function that extracts values from the theme.
+ * Allows passing multiple strings as props to dig into nested values.
+ * This is intended for use in styled-comopnents's interpolation functions as
+ * an easier way to get properly typed values. Examples:
+ *
+ * instead of:
+ *
+ * styled.div`
+ *   margin-top: ${({theme}: {theme: DefaultTheme}):string => theme.ws.xsmall}
+ * `
+ *
+ * use instead:
+ *
+ * styled.div`
+ *   margin-top: ${fromTheme('ws', 'xsmall')}
+ */
+
+export const fromTheme = (...levels: string[]) => (
+  ({ theme }: { theme: DefaultTheme }): string => {
+    const propList = [...levels];
+    let currentLevel = theme;
+    while (propList.length > 0) {
+      const nextLevel = propList.shift();
+      if (nextLevel in currentLevel) {
+        if (typeof currentLevel[nextLevel] === 'string') {
+          return currentLevel[nextLevel] as string;
+        }
+        currentLevel = currentLevel[nextLevel] as Record<string, unknown>;
+      } else {
+        break;
+      }
+    }
+    return null;
+  });
+
+/**
+ * A set of enums corresponding to the different colors exposed by the Mark One
+ * Theme.
+ *
+ */
+export enum VARIANT {
+  BASE = 'base',
+  INFO = 'info',
+  POSITIVE = 'positive',
+  PRIMARY = 'positive',
+  NEGATIVE = 'negative',
+  SECONDARY = 'base',
+  DANGER = 'negative',
+  DEFAULT = 'base',
+  LIGHT = 'light',
+  DARK = 'dark',
+}
```

Comment:
I'm pretty sure that `.d.ts` files don't actually get compiled into javascript -- they're just used in the context of the typescript compiler. Since we do use these actual `VARIANT` values in code, we want them to be in the final bundle. This could have stayed in the `MarkOneTheme` file, but since it seemed more like a utility thing, I figured it made sense here.